build/utils.ps1

```diff
@@ -0,0 +1,131 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+if (Test-Path function:\exec) {
+    # already included
+    return
+}
+
+# Fix temp path for non-windows platforms if missing
+if (!(Test-Path env:\TEMP)) {
+    $env:TEMP = [System.IO.Path]::GetTempPath()
+}
+
+function Test-BuildCompiler {
+    # By default we want to build the compiler
+    # If no env var is defined we default to true
+    if (!(Test-Path env:\ENABLE_COMPILER)) {
+        $true
+    }
+    else {
+        # The env var exists, we use its value
+        # Coerce falsy values and negate to keep the defalt $true
+        $env:ENABLE_COMPILER -ne $false
+    }
+}
+
+function Test-BuildVsix {
+    ($Env:ENABLE_VSIX -ne "false") -and $IsWindows
+}
+
+function Test-BuildLlvmComponents {
+    ((Test-Path env:\ENABLE_LLVM_BUILDS) -and ($env:ENABLE_LLVM_BUILDS -eq $true))
+}
+
+# Utilities
+function exec {
+    [CmdletBinding()]
+    param(
+        [Parameter(Mandatory = $true)]
+        [scriptblock]$cmd,
```

Comment:
[style nit]

```suggestion
        [scriptblock]$Command,
```

Sources:
- https://docs.microsoft.com/en-us/powershell/scripting/developer/cmdlet/activity-parameters?view=powershell-7.1
- https://docs.microsoft.com/en-us/powershell/scripting/developer/cmdlet/strongly-encouraged-development-guidelines?view=powershell-7.1#use-pascal-case-for-parameter-names